kalite/distributed/static/js/distributed/base/handlebars.js

```diff
@@ -113,4 +113,8 @@ Handlebars.registerHelper("ifObject", function(candidate, options){
     }
 });
 
+Handlebars.registerHelper("current_language_is", function(lc, options) {
+    return (window.sessionModel.get("CURRENT_LANGUAGE") == lc)?options.fn(this):options.inverse(this);
```

Comment:
A little nitpick:  Any reason for using the `?:` operator here @mrpau-eugene?  For consistency and readability, can you modify this to use `if` as per line 109 above?